libraries/keycloak-user-storage-rest/pom.xml

```diff
@@ -0,0 +1,72 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>no.fdk.keycloak</groupId>
```

Comment:
De andre komponentene har groupId no.dcat.
Burde ikke denne være lik?

Reply:
Det skulle være lik, men dcat er feil. Vi har allerede groupId no.fdk.imcat og user-api har no.fdk

God spørsmål er - skulle groupId være no.fdk, eller no.fdk.keycloak som vi ser - user-api og imcat er allerede forskjellige 😄 

https://maven.apache.org/guides/mini/guide-naming-conventions.html
groupId uniquely identifies your project across all projects. A group ID should follow Java's package name rules.

Reply:
Ja, det har blitt litt rot her. Dcat blir jo feil på mange av komponentene, siden vi nå lager tjenster for andre ting enn det som følger DCAT-standarden. Vi burde nok hatt en gjennomgang av både pakkenavn og group-id i Maven for alle modulene. Kan være aktuelt i forbundelse med refaktorering til separate repositoryer. Arkitekten bør nok se på det.

Reply:
Vi kan endre alle, hvis vi vil. Men de nye skulle minst være bedre.